package.json

```diff
@@ -118,39 +116,32 @@
     "@types/koa": "^2.13.6",
     "@types/koa-bodyparser": "^4.3.10",
     "@types/node": "^20.2.5",
-    "@typescript-eslint/eslint-plugin": "^5.59.2",
+    "@typescript-eslint/eslint-plugin": "^5.61.0",
     "@typescript-eslint/parser": "^5.59.2",
-    "acorn": "^8.8.2",
     "benchmark-suite": "^0.1.8",
     "commander": "^10.0.1",
     "commitizen": "^4.3.0",
     "commitlint": "^17.6.3",
     "conventional-github-releaser": "^3.1.5",
     "eslint": "^8.40.0",
-    "eslint-config-airbnb": "^19.0.4",
     "eslint-config-airbnb-base": "^15.0.0",
     "eslint-config-airbnb-typescript": "^17.0.0",
     "eslint-config-prettier": "^8.8.0",
     "eslint-plugin-import": "^2.27.5",
     "eslint-plugin-json": "^3.1.0",
```

Comment:
All these eslint dependencies are redundant ?

Reply:
Yes checked without them also eslint is working